Sources/SwiftBSON/BSONBinary.swift

```diff
@@ -89,13 +89,13 @@ public struct BSONBinary: Equatable, Hashable {
         self = try BSONBinary(buffer: buffer, subtype: subtype)
     }
 
-    internal init(bytes: [UInt8], subtype: Subtype) throws {
+    public init(bytes: [UInt8], subtype: Subtype) throws {
```

Comment:
mind adding some basic docstrings to these? I think they would be pretty similar to the ones for the init that takes a `Data` (same error case re: incompatible subtype).

Reply:
done